lib/Wikia/src/Vignette/UrlGenerator.php

```diff
@@ -33,34 +33,34 @@ class UrlGenerator {
 	const REVISION_LATEST = 'latest';
 
 	/** @var UrlConfig */
-	private $config;
+	protected $config;
```

Comment:
You see? This is exactly why `private` parts should have been `protect`-ed in the first place. :-)